### PR TITLE
[DC-1743] Add customer reply form.

### PR DESCRIPTION
**Pull request by lxol**

### CODE REVIEW

File: app/controllers/ReplyController.scala [hunk 1]
```diff
@@ -0,0 +1,91 @@
...
+                                  twoWayMessageConnector: TwoWayMessageConnector)
+  extends FrontendController with AuthorisedFunctions with I18nSupport {
+
+  val form: Form[ReplyDetails] = formProvider()
+
+  def onPageLoad(queue: String, replyTo: String): Action[AnyContent] = Action.async {
+    implicit request =>
+      authorised(Enrolment("HMRC-NI")).retrieve(Retrievals.email) {
+        case email => {
+          Future.successful(Ok(reply(queue, replyTo, appConfig, form, ReplyDetails( ""))))
```
Comment (sammhmrc) [inline app/controllers/ReplyController.scala]: You could use the ReplyDetails object to pre-populate the form. Check with Jerome.

File: app/forms/ReplyFormProvider.scala [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+import play.api.data.Form
+import play.api.data.Forms._
+
+class ReplyFormProvider @Inject() extends FormErrorHelper with Mappings {
+  private val SUBJECT_MAX_LENGTH = 65
+
+  def apply(): Form[ReplyDetails] =
+    Form(
+      mapping(
+        "content" -> text()
```
Comment (sammhmrc) [inline app/forms/ReplyFormProvider.scala]: Validation against empty strings?

File: conf/messages.en [hunk 3]
```diff
@@ -53,5 +53,13 @@ enquiry.confirmEmail = Confirm email
 enquiry.whatis = Type of the question
 enquiry.question = What is your question?
 
+
+reply.title = Reply to a secure answer
```
Comment (sammhmrc) [inline conf/messages.en]: I personally don't think these need to be externalised and would be best baked straight into the templates themselves. What do you think?

File: app/forms/ReplyFormProvider.scala (hunk 2 above)
Comment (lxol) [reply app/forms/ReplyFormProvider.scala]: yes

File: app/controllers/ReplyController.scala (hunk 1 above)
Comment (lxol) [reply app/controllers/ReplyController.scala]: The form contains only one field - the input for the customer reply. I don't think we can read customer's mind and prepopulate his/her reply.

File: conf/messages.en (hunk 3 above)
Comment (lxol) [reply conf/messages.en]: I just followed the convention of the  enquiry form that we had already

Comment (lxol) [reply conf/messages.en]: also it will help if we need to support Welsh in the future

**Pull request merged**